objects/src/block/header.rs

```diff
@@ -18,13 +19,13 @@ use crate::utils::serde::{
 /// - note_root is a commitment to all notes created in the current block.
 /// - batch_root is a commitment to a set of transaction batches executed as a part of this block.
 /// - proof_hash is a hash of a STARK proof attesting to the correct state transition.
-/// - version specifies the version of the protocol.
-/// - timestamp is the time when the block was created.
+/// - timestamp is the time when the block was created, in seconds since UNIX epoch.
```

Comment:
I would probably add a sentence here saying something like: "The current representation (4 bytes) is sufficient for timestamps up to year 2106." (as you've mentioned in your comment above).